go/interpreter/lfvm/converter.go

```diff
@@ -100,16 +98,13 @@ func (b *codeBuilder) toCode() Code {
 	return b.code[0:b.nextPos]
 }
 
-func convert(code []byte, with_super_instructions bool) (Code, error) {
+func convert(code []byte, with_super_instructions bool) Code {
 	res := newCodeBuilder(len(code))
 
 	// Convert each individual instruction.
 	for i := 0; i < len(code); {
 		// Handle jump destinations
 		if code[i] == byte(vm.JUMPDEST) {
```

Comment:
This change removes the check, but since length returns nextPos, nextPos is monotonic, and the relationship between nextPos and i depend on the code structure; I guess that the error can still happen given a malformed program as input. 

In this case, the virtual machine may even panic if nextPos grows > len(code)`.

I don't think the vm should ever crash, even with invalid/bogus/malformed/corrupted/malicious code. Can we guarantee this after removing this check?